### DISCUSSION TITLE
Looking to join a team to develop an NPC demo

### DISCUSSION THREAD

**Discussion by samgriek** [daveshap/ACE_Framework#20] [General]
Let me know if anyone would like to start to form another scrum team to work on the actual ACE Framework backlog.  I believe this is what Dave had said in his video.  I've implemented a hello world base don the ACE Framwork paper but want to help work on the main backlog. 

As far as my background, I've been coding since 2003 mostly in data, big data, data processing (ETL, validation, reporting).  Really too many technologies to keep this short.  For the last 6 years I've focused on Python in both data and APIs (GraphQL and REST). And this year I've been building fullstack apps with Quasar and FastAPI.

I'm interested in several use-cases with this framework but I would like to join a team to work on the main backlog...

**Comment by daveshap** [comment]
One point of clarification, we won't have one main backlog, but rather several parallel projects that we'll help with. The "main backlog" is primarily the main reference architecture e.g. single user assistant or something like that. For some parallel projects, here are some ideas:

1. Robot like Wall-E
2. NPCs in procedurally generated 2D game world
3. NPCs in Unity or Unreal
4. Autonomous Employee (e.g. able to do work, primary interface is Discord, Teams, or Slack)
5. Research assistant
6. Automated politician (??? someone suggested this)

**Reply by samgriek** [reply]
Gotcah, my mistake.

**Reply by daveshap** [reply]
No worries, we're still working on organizing everything

**Reply by ghost** [reply]
Tbc, it wasn't me but I really do think automated politician is one that those with more knowledge than I seriously consider for their project. 

You can't offer an alternative for comparison until you have one and I imagine many would be quite ready to accept this idea in our modern times of difficulty, to somewhat understate, I'm assuming that an automated politician would demonstrate itself to be a vastly superior option to the crop of idiots we elect worldwide.

There will be no real political change for the better until a better alternative is offered. Tell me people are not crying out worldwide for equal treatment. Our politicians will never and I mean never offer even such a basic consideration. How many times have we heard the phrase '.... but there was no political will'?

There will be no environmental change for the better until there is a political will!

Without such a political will I fear and am confident in my assessment that we will not weather the storm. Politicians, with their assumed flags and armies, are the problem.

**Comment by hexadecimal-REBOOT** [comment]
I'm willing to join your team but I'm interested in NPCs I have a idea on dialogue and I don't mean using the AI to generate the text for the NPC

**Comment by samgriek** [comment]
> I'm willing to join your team but I'm interested in NPCs I have a idea on dialogue and I don't mean using the AI to generate the text for the NPC

@hexadecimal-REBOOT 

The NPC case is compelling.  I think it's a great one to start with.  What do you think about a video call to organize a bit?  Reach out to me on LinkedIn and we can set something up.  Depending on our combined skill set and/or interests, we can recruit some more folks from my previous teams... <link deleted> (My LinkedIn Profile).

**Reply by daveshap** [reply]
Consider starting with AI Town https://github.com/a16z-infra/ai-town

**Reply by hexadecimal-REBOOT** [reply]
Sounds good I added you and I will send you a message so we can get the ball rolling

**Reply by daveshap** [reply]
Glad to see some teams forming. Remember that my core team will be available to provide guidance if you need. Also, please post your results in [Demos & Implementations](https://github.com/daveshap/ACE_Framework/discussions/categories/demos-implementations) once you have them!

**Reply by samgriek** [reply]
@hexadecimal-REBOOT - Found you I believe and also sent you an invite to my network as I don't see your invite for some reason.  Anyway, let me know if you didn't get an invite either and we'll figure out another way to do connect...

**Comment by hexadecimal-REBOOT** [comment]


**Reply by daveshap** [reply]
Just an FYI publicly posted messages should never be considered privileged or confidential. I cannot endorse or support any claims of confidentiality on this repo

**Comment by KenjiPcx** [comment]
Hey @samgriek, I'm Kenji, I had a post here to form a team to build autonomous employee workforce agents, but I saw this video https://www.youtube.com/watch?v=5Zj_zstLLP4 and I think AI town would be a good interface to interact with it, would you be happy to build something together? 

I'm a student so don't have much long term professional work experience, but I have done a ton of other gigs/internships as a student, here is my linkedIn https://www.linkedin.com/in/kenjiphang/

**Comment by GeorgiaPhillips1008** [comment]
Hi @samgriek I'd also be very interested in working with you on the NPC case. I'm a software engineer mostly working with a Java/JS tech stack now, but did a lot of internships/projects with Python during undergrad. Here is my linkedIn: www.linkedin.com/in/georgia-phillips-8a196a17b. Let me know if you'd like to plan a time to meet up!

**Comment by Rathi96** [comment]
Looking to enhance your product delivery process? Check out this resource on managing spilled stories for better predictability here. It’s a game-changer! https://www.kaiburr.com/blog/how-to-improve-predictability-of-product-delivery-by-managing-spilled-stories/
